tools/RepositoryManagement/DevHome.RepositoryManagement/ViewModels/RepositoryManagementItemViewModel.cs

```diff
@@ -370,7 +398,7 @@ private void SendTelemetryAndLogError(string operation, Exception ex)
         _log.Error(ex, string.Empty);
     }
 
-    private async Task CloneLocationNotFoundNotifyUser()
+    private async Task CloneLocationNotAndFoundNotifyUserAsync()
```

Comment:
slight typo on this one, looks like the words `And` and `Found` need to be switched. But thinking about the purpose for this method more maybe `ShowCloneLocationNotFoundDialogAsync`. Either of the two works but the `And` and the `Found` switch caught my eye initially.

Reply:
I like that idea.  Changed.